### DISCUSSION TITLE
Using Filter in DataTable on a column with DataType date including hours and minutes

### DISCUSSION THREAD

**Discussion by Lyustar** [primefaces/community#1115] [PrimeReact]
Hello!
I'm using a filter in a table on a column with data type date. In my case, the date contains hours and minutes. 
The standard filter for dateBefore and dateAfter comparisons works correctly. But for comparisons dateIs and dateIsNot it produces an erroneous result, because does not take into account hours and minutes. 

In the new version, could you use value.getTime() and filter.getTime() for dateIs and dateIsNot comparisons in the api.esm.js file instead of the values.toDateString() and filter.toDaeString()?

Thank you for considering my question.

**Comment by melloware** [comment]
I would open a GitHub ticket with StackBlitz reproducer showing the issue...
